cypress/objects/loginPageObject.js

```diff
@@ -0,0 +1,22 @@
+class LoginPage {
+
+    visit() { cy.visit('/') };
+    getUserValidationIndicator = () => cy.get('.Select.not-valid');
```

Comment:
You do not return this. Does it still work?

Reply:
arrow types function - always returns a value Don't need to return it again.